test/unit-test/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+# Include filepaths for source and include.
+include( ${MODULE_ROOT_DIR}/mqttFilePaths.cmake )
```

Comment:
I think you should include both `mqttAgentFilePaths.cmake` and original `mqttFilePaths.cmake` here so that you can use variables declared in those files.